doc/tutorials/acrn_on_qemu.rst

```diff
@@ -217,61 +217,26 @@ Install ACRN Hypervisor
 Bring-Up User VM (L2 Guest)
 ***************************
 
-1. Build the ACRN User VM kernel.
+1. Build the User VM disk image (``UserVM.img``) following :ref:`build-the-ubuntu-kvm-image` and copy it to the ACRNSOS (L1 Guest).
+   Alternatively you can also use ubuntu ISO image.
```

Comment:
```suggestion
   Alternatively you can also use an `Ubuntu Desktop ISO image <https://ubuntu.com/#download>`_.
   Rename the downloaded ISO image to ``UserVM.iso``.
```
We recommend using a desktop image, right?  And they'll need to rename the ISO image for the rest of these instructions to work, right?

Reply:
Done!